karma.conf.js

```diff
@@ -1,30 +1,21 @@
 process.env.ethTest = 'TransactionTests'
 
-module.exports = function (config) {
+module.exports = function(config) {
   config.set({
     browserNoActivityTimeout: 60000,
     frameworks: ['browserify', 'detectBrowsers', 'tap'],
-    files: [
-      './test/api.js',
-      './test/transactionRunner.js'
-    ],
+    // the official transaction's test suite is disabled for now, see https://github.com/ethereumjs/ethereumjs-testing/issues/40
+    files: ['./test-build/test/api.js', './test-build/test/fake.js'],
     preprocessors: {
-      'test/*.js': ['browserify', 'env']
+      './test-build/**/*.js': ['browserify']
     },
     singleRun: true,
-    plugins: [
```

Comment:
These aren't needed anymore? in that case maybe we can also drop their dependencies.

Reply:
It's just the declaration that isn't needed. I had to update karma, and the newer version autoloads plugins AFAIK.